ColorPicker/SwatchButton.js

```diff
@@ -37,10 +37,10 @@ const SwatchButtonBase = kind({
 		 *
 		 * The value should take the format of a HEX color. Ex: `#ffcc00` or `#3467af`
 		 *
-		 * @type {String}
+		 * @type {Object}
```

Comment:
`Object` is right? because above description " The value should take the format of a HEX color. Ex: `#ffcc00` or `#3467af`"
I confused.

Reply:
I found the reason through the unit test.